nemo/collections/asr/models/rnnt_models.py

```diff
@@ -301,9 +345,16 @@ def transcribe(
                 self.encoder.unfreeze()
                 self.decoder.unfreeze()
                 self.joint.unfreeze()
+                if hasattr(self, 'ctc_decoder'):
+                    self.ctc_decoder.unfreeze()
         return hypotheses, all_hypotheses
 
-    def change_vocabulary(self, new_vocabulary: List[str], decoding_cfg: Optional[DictConfig] = None):
+    def change_vocabulary(
+        self,
+        new_vocabulary: List[str],
+        decoding_cfg: Optional[DictConfig] = None,
+        ctc_decoding_cfg: Optional[DictConfig] = None,
+    ):
```

Comment:
add those args to comments. This should be done for all changes in the function argument lists.